public/app/plugins/datasource/prometheus/partials/config.html

```diff
@@ -48,3 +48,23 @@
   </div>
 </div>
 
+<h3 class="page-heading">Misc</h3>
+<div class="gf-form-group">
+  <div class="gf-form-inline">
+    <div class="gf-form max-width-30">
+      <span class="gf-form-label width-13">Custom query parameters</span>
+      <input
+        type="text"
+        class="gf-form-input gf-form-input--has-help-icon"
+        ng-model="ctrl.current.jsonData.customParameters"
+        spellcheck="false"
+        placeholder="max_source_resolution=5m&timeout=10"
```

Comment:
Can we prefix the placeholder with `Example: `? Otherwise it might look like the placeholder is the default.

Reply:
sure !